### PR TITLE
feat: 2fa activation supporting styles

### PR DESCRIPTION
**Pull request by gorjan5sk**

### CODE REVIEW

File: src/css/utils/_color.scss [hunk 1]
```diff
@@ -68,6 +72,18 @@
...
+.color-white {
+  color: var(--sn-stylekit-white);
+}
+
+.color-dark-red {
+  color: var(--sn-stylekit-dark-red);
+}
+
+.stroke-info {
+  stroke: var(--sn-stylekit-info-color);
```
Comment (antsgar) [inline src/css/utils/_color.scss:84]: Could we maybe use stroke-current and use text color utilities, like Tailwind https://tailwindcss.com/docs/stroke?

File: src/css/utils/_position.scss [hunk 2]
```diff
@@ -10,3 +10,11 @@
   @extend .top-0;
   @extend .left-0;
 }
+
+.top-neg-10 {
```
Comment (antsgar) [inline src/css/utils/_position.scss:14]: For negative positioning the Tailwind naming convention would be -top-10 and -left-51, check here https://tailwindcss.com/docs/top-right-bottom-left

File: src/css/utils/_typography.scss [hunk 3]
```diff
@@ -42,6 +42,11 @@
   line-height: 1.75rem;
 }
 
+.text-lg-sm-lh {
```
Comment (antsgar) [inline src/css/utils/_typography.scss:45]: Is a different line height 100% necessary or can we go for changing it to the default line height for consistency? I guess it depends on whether this specific styling is used more than one time or not

**Review by antsgar — APPROVED** [review]

File: src/css/utils/_color.scss (hunk 1 above)
Comment (gorjan5sk) [reply src/css/utils/_color.scss:84]: Ah, this is a remnant. I was trying to use it on SVG, didn't work. Fix is in: 
https://github.com/standardnotes/StyleKit/pull/44 .

File: src/css/utils/_position.scss (hunk 2 above)
Comment (gorjan5sk) [reply src/css/utils/_position.scss:14]: Fixed in above-mentioned PR.

File: src/css/utils/_typography.scss (hunk 3 above)
Comment (gorjan5sk) [reply src/css/utils/_typography.scss:45]: I adjusted with padding. Fixed in above-mentioned PR.

**Pull request merged**